ml_logger/ml_logger.py

```diff
@@ -447,11 +447,11 @@ def log_module(self, namespace="modules", fmt="04d", step=None, **kwargs):
             self.flush()
             self.step = step
 
-        for k, m in kwargs.items():
+        for var_name, module in kwargs.items():
             # todo: this is torch-specific code. figure out a better way.
-            ps = {k: v.cpu().detach().numpy() for k, v in m.state_dict().items()},
+            ps = {k: v.cpu().detach().numpy() for k, v in module.state_dict().items()}
             # we use the number first file names to help organize modules by epoch.
-            path = os.path.join(namespace, f'{k}.pkl' if self.step is None else f'{step:{fmt}}_{k}.pkl')
+            path = os.path.join(namespace, f'{var_name}.pkl' if self.step is None else f'{step:{fmt}}_{var_name}.pkl')
             self.log_data(path=path, data=ps)
```

Comment:
change names to avoid dict comprehension local variable from being the same as loop variable.

this shouldn't be a problem in python3, but changed it anyways.